### DISCUSSION TITLE
[V5] Why _method: 'DELETE' is not working? (in my case)

### DISCUSSION THREAD

**Discussion by rbartholomay** [adonisjs/core#3549] [Help] [answered]
Hi there,

i enable ``allowMethodSpoofing: true,`` inside app config. All runs fine, but on a page i have a link which should delete a post. So i try
```
<a href="{{route('PostsController.destroy', { id: row.id }, {qs: { _method: 'DELETE'}} ) }}"> Delete post </a>
```

The routes is defined as ``Route.resource('/posts', 'PostsController')`` 

When i click the link, it will jump into the show method of the controller. Why?

**Comment by galaczi** [comment, marked as answer]
You have to use a form and POST method.
